init.sh

```
@@ -34,9 +34,16 @@ if is_local "$REG_OCPHOST"; then
 	#echo run local REM_REG_PATH=$REM_REG_ROOT
 else
     SUB_PATH=${REG_ROOT#$HOME/}
-    KNI_HOME=$(ssh ${REG_KNI_USER}@${REG_OCPHOST} "pwd")
-    REM_REG_ROOT=$KNI_HOME/$SUB_PATH
-    #echo run remte REM_REG_ROOT=$REG_PATH
+    if [[ "$SUB_PATH" != "$REG_ROOT" ]]; then
+        KNI_HOME=$(ssh ${REG_KNI_USER}@${REG_OCPHOST} "pwd")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)